templates/vue/src/components/TapestrySidebar.vue

```diff
@@ -28,11 +32,15 @@
       <header class="sidebar-header">
         <h1 ref="info" data-name="info" class="content-title">{{ node.title }}</h1>
         <div class="button-container">
-          <b-button v-if="node.accessible || canEdit" @click="viewNode">
+          <b-button
```

Comment:
Same thing here. Is there anyway to combine this with the conditions in the regular edit button?